vault-cfn-template/readme.adoc

```diff
@@ -0,0 +1,3 @@
+This template has AMI mappings to all regions, with parameters options to select VPCid and subnets where you want your instances to be created. The default instance type is m4.large
```

Comment:
Add a `quick-deploy/cloudformation/readme.adoc` and add details about how to run this CFN template. Ideally provide a fully functional CLI that developers can copy/paste to get this going.